Source/missiles.cpp

```diff
@@ -904,6 +888,26 @@ bool Plr2PlrMHit(int pnum, int p, int mindam, int maxdam, int dist, int mtype, b
 		if (blk > 100) {
 			blk = 100;
 		}
+		switch (missiledata[mtype].mResist) {
+		case MISR_FIRE:
+			resper = plr[p]._pFireResist;
+			break;
+		case MISR_LIGHTNING:
+			resper = plr[p]._pLghtResist;
+			break;
+		case MISR_MAGIC:
+		case MISR_ACID:
+			resper = plr[p]._pMagResist;
+			break;
+		default:
+			resper = 0;
+			break;
+		}
+		if (resper <= 0 && blkper < blk) {

```

Comment:
I found an interesting discrepancy here. Players can't block PvP spells if they have nonzero resistance. I assume this has something to do with the fact that Hellfire didn't officially support multiplayer, but I kept the behavior the same so I could highlight this fact.

Reply:
Feel free to unify if you like.